salad/operator/scripts/wait_for_epoch_transition.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+
+const Web3 = require('web3');
+const {Enigma} = require('enigma-js/node');
+
+const main = async () => {
+    const provider = new Web3.providers.WebsocketProvider(`ws://${process.env.ETH_HOST}:${process.env.ETH_PORT}`);
+    const web3 = new Web3(provider);
+
+    let enigmaHost = process.env.ENIGMA_HOST || 'localhost';
+    let enigmaPort = process.env.ENIGMA_PORT || '3333';
+    const enigmaAddr = process.env.ENIGMA_CONTRACT_ADDRESS;
+    const enigmaTokenAddr = process.env.ENIGMA_TOKEN_CONTRACT_ADDRESS;
+
+    const enigma = new Enigma(
+        web3,
+        enigmaAddr,
+        enigmaTokenAddr,
+        'http://' + enigmaHost + ':' + enigmaPort,
+        {
+            gas: 4712388,
+            from: web3.eth.accounts[0],
+        },
+    );
+
+    const enigmaContract = enigma.enigmaContract;
+    console.error('Waiting for an epoch to pass after the contract deployment');
```

Comment:
You sure you want to leave it as console.error?

Reply:
Generally i like printing things to stderr rather than stdout, since this is logging, and not real "program output". If you think we should do the opposite here then i don't mind.

Reply:
I mention it because the filebeat in k8s collects logs from stdout IIRC, so I'm not sure if these will be available in the kibana.

Reply:
Yeah, looks like you're right. i'll modify that